webpack.config-test.js

```diff
@@ -36,14 +36,18 @@ config.optimization.splitChunks = false;
 config.plugins = [
 	new VueLoaderPlugin(),
 
+	new webpack.DefinePlugin({
+		__VUE_OPTIONS_API__: JSON.stringify(true),
```

Comment:
I think I'd prefer to just hard code "true" or "false" here and avoid the JSON call